.github/workflows/pull_request.yml

```diff
@@ -186,7 +186,7 @@ jobs:
 
       - name: Build universe-core
         working-directory: ./universe
-        run: rm -r ~/.m2/repository/org/corfudb && ./gradlew clean publishToMavenLocal
+        run: rm -rf ~/.m2/repository/org/corfudb && ./gradlew clean publishToMavenLocal
```

Comment:
Could you please explain why it needs to be forced? as the similar command in compatibility test job works fine with just -r

Reply:
In my case, my PRs seemed to fail often with "no such directory" on ~/.m2/repository/org/corfudb. 
The -f option should address that I believe.

Reply:
Gotcha, make sense, thanks!